### ISSUE TITLE
Make `name=` filter work

### ISSUE THREAD

**Issue by ikapelyukhin** [SUSE-Enceladus/public-cloud-info-client#23]
Currently `name=` filter doesn't seem to work, e.g.`pint google images --filter name=suse-cap` or `pint google images --filter name=foo` produces unexpected output.

**Comment by KeithMnemonic** [comment]
The man page seems to state name= is not a valid option.

Valid filter values for images are:

              deletedon>, deletedon=, deletedon<,
              deprecatedon>, deprecatedon=, deprecatedon<,
              id=,
              name~, name!, name%,

does name~ if you replace it instead of name=

**Comment by ikapelyukhin** [comment]
@KeithMnemonic no, exact match is required. See conversation on !7 PR in brewmaster repo for context. At the very least pint client should die and tell you that `name=` is not a valid filter.
